build.gradle

```diff
@@ -3,7 +3,6 @@ plugins {
     id 'checkstyle'
     id 'pmd'
     id 'jacoco'
-    id "net.linguica.maven-settings" version "0.5"
```

Comment:
denne trenger jeg for at det skal funke lokalt hos meg.

Reply:
Ok, legger den inn og merger